manifests/profile/postgres.pp

```diff
@@ -1,4 +1,4 @@
-# @summary Apply this class to a PE-managed postgres instance to allow access from telegraf
+# @summary This class is deprecated.  Please use the Puppet_metrics_dashboard::Profile::Master::Postgres_access.
```

Comment:
See my comment that I added before starting this review regarding my thoughts on this class.

Reply:
I put a hard fail in the deprecated class